### PR TITLE
CompanyProfile should display Edit/Delete profile and Manage Opportunities…

### PR DESCRIPTION
**Pull request by philezemonye**
CompanyProfile component should display Edit/Delete profile and Manage Opportunities button.

### CODE REVIEW

File: client/src/components/CompanyProfile.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  handleClickEdit = () => {};
+
+  handleClickDelete = () => {};
+
+  render() {
+    return (
+      
+      <div class="ui segment">
+        <Container>
+          <Container text style={{ marginTop: "5em" }} border={{}}>
```
Comment (khaledkz) [inline client/src/components/CompanyProfile.js]: I do not think it's great to do inline style? the best here to add classname and style it using this class name, but not Container can take class name?

File: client/src/components/CompanyProfile.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
...NYDK4oIFob1oIoygqVhPaCikGBxWA+mKCQICOuBFKUEEWE9iKKYICSsB1CUE8SE9YoV5QRRYb1CRUHBCR1ZblivSFFSEBfWK1CUFESG9bIVRQWhYb1MRVlBbliPR5airCA62J2hKCzIDOvxSVaUFuSF9VJIVBQXZIX10khSFBdkhfVSSVCUF+SE9dJhK1YQDB6qhWG9rG55ihUEGWG9PFiKNQQZYb1MGIpVBPkBjGQGFesIMsJ62Qwo1hFkhfWy6VWsJBiE9XoDGMn0KL5VEkwMYKT336G4CD7algd4LZzUAEYyU7qa3vIS/MK14F8b54b18gkue38/yVrnkP4F3FzxS+c5rPR/mBDWy6Pr//Ct1m9pSlgvB/3pIimsl47+lB8EMLBzhYHLNtm5wsKld3JYLwUTt0+0Y+SryyZugXPCelxsLGPQsN43otMWI0tReWE9DkaWE3PDesNYWRIWq6xnZllfqrKenUcztD9QtSQ7j9eEKusZekQqU1nP0mNukcp6lqIKIpX1TMVNaADjJLOffWxFhgQq6xmLfdFuyucKY9E9fGU9a/FLeGU9cxFa2kXpXGEuBo2urGcvyg6urGfwdQRsZT2Dr5QYCOv1AVCEBjBMvtqFDOvZfD2PHlkQ1rP5iiUwrGf0NVlcWM/qq85f5KjssJ7V19VhYT2zJQdQc4XdshGgsJ7h0h/0gLwAhuHyLZgAhuUSPJCwnukySoiwnulSWIiwnu1yZoAAhvGSdOVhPetlBWm75LnCemnI4rnCfHlPGsBIDeupCCbVoC0N6+nUoE2oI1wa1tOqI8yvBV0awFCrBc2u510a1tOr582tyU5bpIb1FGuy8+rqB2G95ACGYl191t4IgLCe4t4InP0tEGE9xf0thvcowYT1FPcoGdxnBhTWU9xnZmivIFRYT3GvoIH9nugpzA5gKO731O7Z1fHMGhjWe1putfbs6sO3wT16fBvc48e3wT16fBvc44eewtHNFb4N7vFzBNvgloEN61lEqlqSHdCV9exBT2FxZT1rBPcVwGpJNqCrbIjKesb4OjSEVNazBTmJmMp6tpiu9x+Oao9GiOniL7Q3ulvDPeY3q2YDrujlOI7jOI7jOI7jOI7jOI7jOI5zxPwDKH88KV+5d44AAAAASUVORK5CYII=" size="small" circular background="grey"
```
Comment (khaledkz) [inline client/src/components/CompanyProfile.js]: what does this image refer to??

File: client/src/components/CompanyProfile.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+           <h1>Company Name</h1>
+           <h2>Location</h2>
+           </Divider>
+           
+           <Divider horizontal>
+           <h3>Contact</h3>
+           </Divider>
+     <div>
+           <h4>About Company</h4>
+           <p> Global leader in financial services, offering solutions
```
Comment (khaledkz) [inline client/src/components/CompanyProfile.js]: I do not think you need to write the about company paragraph by yourself, you need to get the data from the API and use that data.

File: server/api/applicantProfile.js [hunk 4]
```diff
@@ -3,74 +3,74 @@ const router = express.Router();
 const applicantProfileDb = require("../services/database/applicantProfile");
 const db = require("../services/database/users");
 const {
-  createApplicantProfile,
+	createApplicantProfile,
```
Comment (DwinaTech) [inline server/api/applicantProfile.js]: Fix the indentation

File: server/api/applicantProfile.js [hunk 5]
```diff
@@ -3,74 +3,74 @@ const router = express.Router();
...
-  applicantProfileDb
-    .getApplicantProfile(id)
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.send(500);
-    });
+	const { id } = req.params;
```
Comment (DwinaTech) [inline server/api/applicantProfile.js]: Here too

File: server/api/applicantProfile.js [hunk 6]
```diff
@@ -3,74 +3,74 @@ const router = express.Router();
...
-    role,
-    email,
-    password,
-    about,
-    city,
-    skills,
-    cvLink,
-    value,
-  } = req.body;
+	const {
```
Comment (DwinaTech) [inline server/api/applicantProfile.js]: Here too

File: server/api/applicantProfile.js [hunk 7]
```diff
@@ -3,74 +3,74 @@ const router = express.Router();
...
-    name,
-    email,
-    password,
-    about,
-    city,
-    skills,
-    cvLink,
-    value,
-  };
+	const applicant = {
```
Comment (DwinaTech) [inline server/api/applicantProfile.js]: Here too

File: server/api/applicantProfile.js [hunk 8]
```diff
@@ -3,74 +3,74 @@ const router = express.Router();
...
-      return createApplicantProfile(userData).then((data) => {
-        const profileId = data[0].applicant_id;
-        return { profileId };
-      });
-    })
-    .then((profileId) => {
-      newApplicantSkills({ profileId, skills });
-    })
-    .then((data) => res.send({ success: true }))
+	db.createUser(applicant)
```
Comment (DwinaTech) [inline server/api/applicantProfile.js]: Here too

File: server/api/cities.js [hunk 9]
```diff
@@ -7,15 +7,15 @@ const cities = require("../services/database/cities");
...
-  cities
-    .getAllCities()
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.status(500);
-    });
+	cities
```
Comment (DwinaTech) [inline server/api/cities.js]: Here too

File: server/api/companies.js [hunk 10]
```diff
@@ -4,49 +4,49 @@ const companyRegister = require("../services/database/companies");
...
-  companyRegister
-    .getAllCompanies()
-    .then(data => {
-      res.send(data);
-    })
-    .catch(err => {
-      console.error(err);
-      res.send(500);
-    });
+	companyRegister
```
Comment (DwinaTech) [inline server/api/companies.js]: Here too

File: server/api/companies.js [hunk 11]
```diff
@@ -4,49 +4,49 @@ const companyRegister = require("../services/database/companies");
...
+			console.error(err);
+			res.send(500);
+		});
 });
 
 router.post("/", (req, res) => {
-  const name = req.body.name;
-  const description = req.body.description;
-  const industry = req.body.industry;
+	const name = req.body.name;
```
Comment (DwinaTech) [inline server/api/companies.js]: Here too

File: server/api/companyProfile.js [hunk 12]
```diff
@@ -6,16 +6,16 @@ const companyProfileDb = require("../services/database/companyProfile");
...
-  companyProfileDb
-    .getCompanyProfile(id)
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.send(500);
-    });
+	const { id } = req.params;
```
Comment (DwinaTech) [inline server/api/companyProfile.js]: Here too

File: server/api/opportunities.js [hunk 13]
```diff
@@ -2,7 +2,7 @@ const express = require("express");
 const router = express.Router();
 const { createOpportunity } = require("../services/database/opportunities");
 const {
-  newOpportunitySkills,
+	newOpportunitySkills,
```
Comment (DwinaTech) [inline server/api/opportunities.js]: Here too

File: server/api/opportunities.js [hunk 14]
```diff
@@ -11,43 +11,43 @@ const {
...
-    contactPerson,
-    telephone,
-    email,
-    city,
-    date,
-    type,
-    skills,
-    company_id,
-  } = req.body;
+	const {
```
Comment (DwinaTech) [inline server/api/opportunities.js]: Here too

File: server/api/opportunitySkills.js [hunk 15]
```diff
@@ -8,16 +8,16 @@ const opportunitySkills = require("../services/database/opportunitySkills");
...
-  const opportunityId = req.body.opportunityId;
-  opportunitySkills
-    .newOpportunitySkills(skills, opportunityId)
-    .then((data) => {
-      res.send({ success: true });
-    })
-    .catch((err) => {
-      res.sendStatus(500).send({ success: false });
-    });
+	const skills = req.body.skills;
```
Comment (DwinaTech) [inline server/api/opportunitySkills.js]: Here too

File: server/api/skills.js [hunk 16]
```diff
@@ -7,15 +7,15 @@ const skills = require("../services/database/skills");
...
-  skills
-    .getAllSkills()
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.SendStatus(500);
-    });
+	skills
```
Comment (DwinaTech) [inline server/api/skills.js]: Here too

File: server/api/status.js [hunk 17]
```diff
@@ -4,19 +4,19 @@ const passport = require("passport");
 
 /** route is /status/ (status is defined as prefix in index.js) */
 router.get("/", (req, res) => {
-  res.send("All good");
+	res.send("All good");
```
Comment (DwinaTech) [inline server/api/status.js]: Here too

File: server/api/status.js [hunk 18]
```diff
@@ -4,19 +4,19 @@ const passport = require("passport");
...
  * This route (/status/protected) is protected. If you GET /api/protected-status without a valid token
  * then it will 403 (forbidden). Check the README for information on how to authenticate and get a token
  */
 router.get(
-  "/protected",
-  passport.authenticate("jwt", { session: false }),
-  (req, res) => {
-    res.send("All good");
-  },
+	"/protected",
```
Comment (DwinaTech) [inline server/api/status.js]: Here too

File: server/api/users.js [hunk 19]
```diff
@@ -6,15 +6,15 @@ const usersDb = require("../services/database/users");
...
-  usersDb
-    .getAllUsers()
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.status(500);
-    });
+	usersDb
```
Comment (DwinaTech) [inline server/api/users.js]: Actually fix all files you changed their indentation 😄 just right click and chose format document.

**Pull request merged**